test/DataSet.test.js

```diff
@@ -1,44 +1,43 @@
 var assert = require('assert');
 var vis = require('../dist/vis');
-var moment = vis.moment;
 var DataSet = vis.DataSet;
 var Queue = vis.Queue;
 // TODO: test the source code immediately, but this is ES6
 
 var now = new Date();
 
 describe('DataSet', function () {
-  it('should work', function () {
-    // TODO: improve DataSet tests, split up in one test per function
+  var data = new DataSet({
+    type: {
+      start: 'Date',
+      end: 'Date'
+    }
+  });
 
-    var data = new DataSet({
-      type: {
-        start: 'Date',
-        end: 'Date'
-      }
-    });
+  // add single items with different date types
+  data.add({id: 1, content: 'Item 1', start: new Date(now.valueOf())});
+  data.add({id: 2, content: 'Item 2', start: now.toISOString()});
+  data.add([
+    //{id: 3, content: 'Item 3', start: moment(now)}, // TODO: moment fails, not the same instance
+    {id: 3, content: 'Item 3', start: now},
+    {id: 4, content: 'Item 4', start: '/Date(' + now.valueOf() + ')/'}
+  ]);
 
-    // add single items with different date types
-    data.add({id: 1, content: 'Item 1', start: new Date(now.valueOf())});
-    data.add({id: 2, content: 'Item 2', start: now.toISOString()});
-    data.add([
-      //{id: 3, content: 'Item 3', start: moment(now)}, // TODO: moment fails, not the same instance
-      {id: 3, content: 'Item 3', start: now},
-      {id: 4, content: 'Item 4', start: '/Date(' + now.valueOf() + ')/'}
-    ]);
+  var sort = function (a, b) {
+    return a.id > b.id;
+  };
+
+  it('can handle multiple data types', function () {
 
     var items = data.get();
```

Comment:
Ensure that var `data` is always in a well-defined state before  running any check on it. This comment applies to *all* usages of `data` in this file.